src/lib/tls/tls13/tls_cipher_state.cpp

```diff
@@ -27,7 +31,7 @@
  *                                     |
  *                                     *
  *                             STATE EARLY TRAFFIC
- * This state is reached by constructing Cipher_State using init_with_psk() (not yet implemented).
+ * This state is reached by constructing Cipher_State using init_with_psk().
```

Comment:
Needs revision: State is reached by initializing with `init_with_psk()` and then advancing with `advance_with_client_hello()`.